mvc/src/main/java/com/interface21/webmvc/servlet/mvc/tobe/AnnotationHandlerMapping.java

```diff
@@ -1,29 +1,96 @@
 package com.interface21.webmvc.servlet.mvc.tobe;
 
-import jakarta.servlet.http.HttpServletRequest;
+import java.lang.reflect.Method;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Stream;
+
+import org.reflections.Reflections;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.util.HashMap;
-import java.util.Map;
+import com.interface21.context.stereotype.Controller;
+import com.interface21.web.bind.annotation.RequestMapping;
+import com.interface21.web.bind.annotation.RequestMethod;
+
+import jakarta.servlet.http.HttpServletRequest;
 
 public class AnnotationHandlerMapping {
 
-    private static final Logger log = LoggerFactory.getLogger(AnnotationHandlerMapping.class);
+	private static final Logger log = LoggerFactory.getLogger(AnnotationHandlerMapping.class);
+
+	private final Object[] basePackage;
+	private final Map<HandlerKey, HandlerExecution> handlerExecutions;
+
+	public AnnotationHandlerMapping(final Object... basePackage) {
+		this.basePackage = basePackage;
+		this.handlerExecutions = new HashMap<>();
+	}
+
+	public void initialize() {
+		Stream.of(basePackage)
+			.map(basePackage -> new Reflections((String)basePackage))
```

Comment:
```suggestion
			.map(Reflections::new)
```
Reflections의 생성자는 Object... 를 인자로 받고 있어서, String으로 형변환 해주지 않아도 괜찮습니다. 그렇다면 위와 같이 간단하게 작성할 수 있겠죠?

Reply:
> Reflections의 생성자는 Object... 를 인자로 받고 있어서, String으로 형변환 해주지 않아도 괜찮습니다. 그렇다면 위와 같이 간단하게 작성할 수 있겠죠?

오호.. 학테에서 문자열만 넣다보니 Object를 넣을 생각을 못했네요~ 감사함다😄